tests/integration/security/sds_tls_origination/main_test.go

```diff
@@ -21,14 +21,34 @@ import (
 	"testing"
 
 	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/echo/common/deployment"
 	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/components/namespace"
 )
 
-var inst istio.Instance
+var (
+	// Namespaces
+	echo1NS    namespace.Instance
+	externalNS namespace.Instance
+
+	// Servers
+	apps deployment.SingleNamespaceView
+)
 
 func TestMain(m *testing.M) {
 	framework.
 		NewSuite(m).
-		Setup(istio.Setup(&inst, nil)).
+		Setup(istio.Setup(nil, nil)).
```

Comment:
it would be great to move this package under security/ to avoid the extra setup costs

Reply:
Moved